infrasim/ipmiconsole/__init__.py

```diff
@@ -152,9 +152,28 @@ def start(instance="default"):
     sdr.parse_sdrs()
     # running thread for each threshold based sensor
     _spawn_sensor_thread()
+    _start_monitor(instance)
     _start_console(instance)
 
 
+def monitor(instance="default"):
+    while True:
```

Comment:
I think polling is a better way to listen to vbmc, otherwise the ipmi-console will keep running only if no connection attempt is made.